roles/irods/tasks/icat.yml

```diff
@@ -51,15 +53,17 @@
     - name: '{{ irods_ssl_certificate_chain_file }}'
       mode: '0644'
   become: true
+  notify: irodsctl_restart 
 
 - name: If missing, build 4096 bit DHparam /etc/irods/{{ irods_ssl_dh_params_file }} file (takes several minues ...)
   ansible.builtin.command:
-    cmd: /bin/openssl dhparam -2 -out /etc/irods/{{ irods_ssl_dh_params_file }} 4096
+    cmd: /bin/openssl dhparam -2 -out /etc/irods/{{ irods_ssl_dh_params_file }} 2048
```

Comment:
Comment still mentions 4096 bits. Why was this downgraded? (Note that 2048 bits will take less time, but does not comply with UMCG encryption policy.)